charactersheet/charactersheet/viewmodels/character/features.js

```diff
@@ -62,6 +65,14 @@ function FeaturesViewModel() {
         }
     };
 
+    self.populateClass = function(label, value) {
+        self.blankFeature().characterClass(value);
+    };
+
+    self.populateClassEdit = function(label, value) {
```

Comment:
Our convention, so far, has been to not add pre-pop to the edit modal. What's the justification for this?